samples/bluetooth/peripheral_hr/sample.yaml

```diff
@@ -28,12 +28,27 @@ tests:
     tags: bluetooth
     sysbuild: true
   sample.bluetooth.peripheral_hr.minimal:
+    harness: bluetooth
+    platform_allow:
+      - qemu_cortex_m3
+    integration_platforms:
+      - qemu_cortex_m3
+    extra_args:
+      - CONF_FILE=prj_minimal.conf
+    tags: bluetooth
+  sample.bluetooth.peripheral_hr.bt_ll_sw_split.minimal:
```

Comment:
Should we add instructions on how to use these configurations in the README.rst? I assume most users are not aware of `west build -T`